apps/learner/src/routes/content/[id]/quiz/+page.svelte

```diff
@@ -31,14 +35,46 @@
       if (currentQuestionIndex < data.questionAnswers.length - 1) {
         currentQuestionIndex++;
         selectedOptionIndex = -1;
+        isFeedbackModalOpen = false;
       } else {
         //TODO: Redirect quiz to completion page
         console.log('Quiz completed!');
       }
     }
   }
+
+  function checkAnswer() {
+    isCorrectAnswer = selectedOptionIndex + 1 === currentQuestion.answer;
+  }
+
+  function showFeedbackModal() {
+    isFeedbackModalOpen = true;
+  }
 </script>
 
+{#snippet modalFeedbackButton(optionIndex: number)}
+  <div class="flex flex-col items-start">
+    <button
+      class={[
+        'gap-x-2.75 flex w-full items-center rounded-2xl bg-lime-200 p-3',
+        optionIndex !== currentQuestion.answer - 1 && '!bg-slate-100',
+      ]}
+    >
+      <span
+        class={[
+          'flex items-center justify-center rounded-lg bg-lime-400 px-2 py-1 text-xs font-semibold text-black',
+          !isCorrectAnswer && '!bg-slate-900 !text-white',
```

Comment:
For consistency, should we update the color of the option to be `bg-lime-400` when user selected the wrong answer? Right now both options are black.